Apps/frontend/cypress/e2e/2-control-panel/reset-button.js

```diff
@@ -0,0 +1,17 @@
+/// <reference types="cypress" />
+
+describe("everything visible on front page", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:5173/");
+  });
+
+  it("displays btn", () => {
+    cy.get(".control-panel").should("be.visible");
+    cy.get(".button-reset").should("be.visible");
+  });
+
+  it("click btn", () => {
+    // We use the `cy.get()` to get the canvas element
+    cy.get("#button-reset").click();
+  });
+});
```

Comment:
file has to be renamed to `reset-button.cy.js` if cypress should find the file automatically

Reply:
I added this,